e2e/tslint.json

```diff
@@ -1,44 +1,55 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-        "tslint:recommended",
-        "tslint-eslint-rules",
-        "tslint-config-prettier"
+  "defaultSeverity": "error",
```

Comment:
changes in `tslint` are mostly changed by linting. real change are lines #50 - #52.

Reply:
Is this the same as our other tslint file? We should probably just have one.